common/src/main/java/net/william278/huskclaims/command/ClaimBlocksCommand.java

```diff
@@ -127,6 +128,42 @@ private void changeClaimBlocks(@NotNull CommandUser executor, @NotNull User user
         );
     }
 
+    private void changeClaimBlocksPlayerGift(@NotNull CommandUser executor, @NotNull User user, long changeBy) {
+        OnlineUser onlineExecuter = (OnlineUser) executor;
```

Comment:
We should do an instanceof check here and send an error_in_game_only message if the executor is not an OnlineUser.

```java
    if (!(executor instanceof OnlineUser online)) {
        // send locale error
        return;
    }
```